invenio_rdm_migrator/streams/records/table_generators/parents.py

```diff
@@ -29,6 +31,20 @@ def generate_parent_rows(parent):
         created=now,
         updated=now,
     )
+    # parent DOI
+    parent_doi = parent["json"].get("pids", {}).get("doi")
+    if parent_doi and parent_doi["identifier"]:
```

Comment:
Can it have a DOI but not its value? The parent DOI is only assigned when a draft gets a doi, thus it will get a value instantly?